### ISSUE TITLE
Updated version

### ISSUE THREAD

**Issue by paulocoutinhox** [Angel1841/Space-Walk#1]
Hi,

There is any updated version?

Thanks.

**Comment by Angel1841** [comment]
No, at least not yet because im currently working on a Twitter clone that you check in my WebProject repository :)
